### PR TITLE
AR inbound#103

### PR DESCRIPTION
**Pull request by BhaagBoseDK**
#103

### CODE REVIEW

File: gui/templates/advanced.html [hunk 1]
```diff
@@ -208,7 +208,7 @@ <h2>Update Local Settings</h2>
           <input style="text-align:center" id="value" type="number" min="1" max="100" name="value" value="{{ settings.value }}">
           {% elif settings.key == 'AR-Time' %}
             <input style="text-align:center" id="value" type="number" min="1" max="60" name="value" value="{{ settings.value }}">
-          {% elif settings.key == 'AR-MaxFeeRate' or settings.key == 'AF-MaxRate' %}
+          {% elif settings.key == 'AR-MaxFeeRate' %}
```
Comment (cryptosharks131) [inline gui/templates/advanced.html:211]: Intentional?

File: gui/templates/home.html [hunk 2]
```diff
@@ -527,10 +529,12 @@ <h2>Update Auto Rebalancer Settings</h2>
       <input id="variance" type="number" min="0" max="100" name="variance">
       <label title="The minutes we should wait after a failed attempt before trying again" for="wait_period">Wait Period (min): </label>
       <input id="wait_period" type="number" min="1" max="100" name="wait_period">
-      <label title="This enables or disables the Autopilot function which automatically acts upon suggestions on this page: /actions" for="autopilot">Autopilot: </label>
+      <label title="This enables or disables the Autopilot function which automatically acts upon suggestions on this page:" for="autopilot">Autopilot: </label>
```
Comment (cryptosharks131) [inline gui/templates/home.html:532]: Intentional?

File: gui/templates/advanced.html (hunk 1 above)
Comment (BhaagBoseDK) [reply gui/templates/advanced.html:211]: The next line has AF-Max and AF-Min fee. Looks like it was redundant here.

File: gui/templates/home.html (hunk 2 above)
Comment (BhaagBoseDK) [reply gui/templates/home.html:532]: I guess /actions was a typo which got copy pasted in other places too.

**Comment by cryptosharks131** [comment]
No typo, those are telling the user to go to the `/actions` page.

This looks good after we revert those changes.

File: gui/templates/home.html (hunk 2 above)
Comment (BhaagBoseDK) [reply gui/templates/home.html]: https://github.com/cryptosharks131/lndg/pull/104/commits/ee94f33a80d62baf834036f02d0fe77943d48d31

**Comment by BhaagBoseDK** [comment]
> No typo, those are telling the user to go to the `/actions` page.
> 
> This looks good after we revert those changes.

https://github.com/cryptosharks131/lndg/pull/104/commits/ee94f33a80d62baf834036f02d0fe77943d48d31

**Pull request merged**